sqlglot/dialects/postgres.py

```diff
@@ -357,6 +357,16 @@ class Parser(parser.Parser):
 
         JSON_ARROWS_REQUIRE_JSON_TYPE = True
 
+        COLUMN_OPERATORS = {
+            **parser.Parser.COLUMN_OPERATORS,
+            TokenType.ARROW: lambda self, this, path: build_json_extract_path(
```

Comment:
this should go in base, not postgres

Reply:
There are some dialects (e.g. DuckDB) where you can do `x -> <json path>`, so the previous approach was valid for those. Should we instead count those that _do_ support it vs those that don't and refactor accordingly? I.e. if there are more in the latter category, have this in base and move the `to_json_path` to dialect.py or something.

Reply:
ok